src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContextSerializer.cs

```diff
@@ -14,6 +14,12 @@ namespace System.ComponentModel.Design
     /// </summary>
     public class DesigntimeLicenseContextSerializer
     {
+        internal const byte BinaryWriterMagic = 255;
+
+        private const string s_enableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization = "System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization";
```

Comment:
Nit: This should not have `s_` prefix to follow the coding conventions.

Also, there is very little value in defining constants for strings that are used just once. It would be better to just pass the string directly as `TryGetSwitch` argument.